Doc/library/importlib.rst

```diff
@@ -355,34 +355,12 @@ ABC hierarchy::
         (note that some of these attributes can change when a module is
         reloaded):
 
-        - :attr:`__name__`
-            The module's fully qualified name.
-            It is ``'__main__'`` for an executed module.
-
-        - :attr:`__file__`
-            The location the :term:`loader` used to load the module.
-            For example, for modules loaded from a .py file this is the filename.
-            It is not set on all modules (e.g. built-in modules).
-
-        - :attr:`__cached__`
-            The filename of a compiled version of the module's code.
-            It is not set on all modules (e.g. built-in modules).
-
-        - :attr:`__path__`
-            The list of locations where the package's submodules will be found.
-            Most of the time this is a single directory.
-            The import system passes this attribute to ``__import__()`` and to finders
-            in the same way as :data:`sys.path` but just for the package.
-            It is not set on non-package modules so it can be used
-            as an indicator that the module is a package.
-
-        - :attr:`__package__`
-            The fully qualified name of the package the module is in (or the
-            empty string for a top-level module).
-            If the module is a package then this is the same as :attr:`__name__`.
-
-        - :attr:`__loader__`
-            The :term:`loader` used to load the module.
+        - :attr:`module.__name__`
```

Comment:
What more should we say about `load_module()` and these attributes?  The preview just says that `load_module()` is deprecated since 3.4 but I think it should have a stronger admonition -- at least a slated removal in 3.14.  Possibly also saying that the attributes other than `__name__` and `__file__` will be removed from the module object in 3.14.

CC: @brettcannon

Reply:
> Possibly also saying that the attributes other than `__name__` and `__file__` will be removed from the module object in 3.14.

FYI, https://docs.python.org/3/deprecations/pending-removal-in-3.15.html#pending-removal-in-python-3-15 says that `__package__` and `__cached__` are due for removal in 3.15, not 3.14. (And that's nothing to do with me -- they said that before this PR! 😉)

Reply:
I'm slightly loath to add too many details about the individual attributes to the docs for `load_module`, since this PR is predicated on the principle that each attribute should have canonical docs in a single location 😉 but I added inline notes for the deprecated attributes in https://github.com/python/cpython/pull/124709/commits/d68fb58cdec8a9e163ab98de88f28c0c0c6ddc28